### ISSUE TITLE
Syntax error in Dockerfile doesn't generate an error, produces useless image

### ISSUE THREAD

**Issue by ecnepsnai** [containers/podman#10627]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

If a Dockerfile has a VOLUME call with a path that doesn't start with a slash, podman build does not return an error but produces a useless image.

Trying to run this image locks up podman. `podman ps` hangs.

**Steps to reproduce the issue:**

Build and run an image with the following dockerfile:

```dockerfile
FROM alpine:3

VOLUME oops

CMD [ "/bin/sh" ]
```
For example:

```
$ podman build -t example .
STEP 1: FROM alpine:3
STEP 2: VOLUME oops
--> a0a4f313979
STEP 3: CMD [ "/bin/sh" ]
STEP 4: COMMIT example
--> c30345b1c16
c30345b1c164a93d4c45097435d84669c5b038e158354194d232f6ed8b6b574a
$ podman run example
[... nothing happens, just hangs ...]
```

**Describe the results you received:**

Podman locks up for the user.

**Describe the results you expected:**

Preferred: podman build fails due to the syntax error
Alternate: the image doesn't lock up podman

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

Reproduced on two systems:

```
Version:      3.1.2
API Version:  3.1.2
Go Version:   go1.16.3
Built:        Wed May 12 12:27:59 2021
OS/Arch:      linux/amd64
```

```
Version:      3.0.2-dev
API Version:  3.0.0
Go Version:   go1.16.1
Built:        Thu May 20 09:22:46 2021
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```yaml
host:
  arch: amd64
  buildahVersion: 1.20.1
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: conmon-2.0.27-2.fc34.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.27, commit: '
  cpus: 4
  distribution:
    distribution: fedora
    version: "34"
  eventLogger: journald
  hostname: OMITTED
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.12.9-300.fc34.x86_64
  linkmode: dynamic
  memFree: 5398458368
  memTotal: 16653533184
  ociRuntime:
    name: crun
    package: crun-0.19.1-3.fc34.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 0.19.1
      commit: 1535fedf0b83fb898d449f9680000f729ba719f5
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +YAJL
  os: linux
  remoteSocket:
    path: /run/user/1000/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    selinuxEnabled: true
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns-1.1.9-1.fc34.x86_64
    version: |-
      slirp4netns version 1.1.8+dev
      commit: 6dc0186e020232ae1a6fcc1f7afbc3ea02fd3876
      libslirp: 4.4.0
      SLIRP_CONFIG_VERSION_MAX: 3
      libseccomp: 2.5.0
  swapFree: 8589930496
  swapTotal: 8589930496
  uptime: 29h 55m 0.66s (Approximately 1.21 days)
registries:
  search:
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - docker.io
  - quay.io
store:
  configFile: /home/ian/.config/containers/storage.conf
  containerStore:
    number: 2
    paused: 0
    running: 0
    stopped: 2
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/ian/.local/share/containers/storage
  graphStatus:
    Backing Filesystem: btrfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 4
  runRoot: /run/user/1000/containers
  volumePath: /home/ian/.local/share/containers/storage/volumes
version:
  APIVersion: 3.1.2
  Built: 1620847679
  BuiltTime: Wed May 12 12:27:59 2021
  GitCommit: ""
  GoVersion: go1.16.3
  OsArch: linux/amd64
  Version: 3.1.2
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-3.1.2-3.fc34.x86_64
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide? (https://github.com/containers/podman/blob/master/troubleshooting.md)**

Not tested with other versions, did review the troubleshooting guide.

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Both machines are physical, amd64.

**Comment by tobwen** [comment]
Hmm, I've tried your example and it here worked.

<details>
  <summary>Click to expand</summary>

``` shell
$ podman --log-level debug run example
INFO[0000] /home/tobwen/podman/usr/local/bin//podman filtering at log level debug
DEBU[0000] Called run.PersistentPreRunE(/home/tobwen/podman/usr/local/bin//podman --conmon /home/tobwen/podman/usr/local/bin//conmon --network-cmd-path /home/tobwen/podman/usr/local/bin//slirp4netns --runtime /home/tobwen/podman/usr/local/bin//crun --storage-driver overlay --storage-opt overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs --log-level debug run example)
DEBU[0000] Using conmon: "/home/tobwen/podman/usr/local/bin//conmon"
DEBU[0000] Initializing boltdb state at /home/tobwen/.local/share/containers/storage/libpod/bolt_state.db
DEBU[0000] Using graph driver overlay
DEBU[0000] Using graph root /home/tobwen/.local/share/containers/storage
DEBU[0000] Using run root /run/user/1000/containers
DEBU[0000] Using static dir /home/tobwen/.local/share/containers/storage/libpod
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp
DEBU[0000] Using volume path /home/tobwen/.local/share/containers/storage/volumes
DEBU[0000] Set libpod namespace to ""
DEBU[0000] Not configuring container store
DEBU[0000] Initializing event backend journald
DEBU[0000] configured OCI runtime kata initialization failed: no valid executable found for OCI runtime kata: invalid argument
DEBU[0000] configured OCI runtime crun initialization failed: no valid executable found for OCI runtime crun: invalid argument
DEBU[0000] configured OCI runtime runc initialization failed: no valid executable found for OCI runtime runc: invalid argument
DEBU[0000] Using OCI runtime "/home/tobwen/podman/usr/local/bin//crun"
INFO[0000] Found CNI network podman (type=bridge) at /home/tobwen/.config/cni/net.d/87-podman.conflist
INFO[0000] Found CNI network cni-podman1 (type=bridge) at /home/tobwen/.config/cni/net.d/cni-podman1.conflist
INFO[0000] Found CNI network cni-podman (type=bridge) at /home/tobwen/.config/cni/net.d/cni-podman.conflist
DEBU[0000] Default CNI network name podman is unchangeable
INFO[0000] Setting parallel job count to 25
INFO[0000] /home/tobwen/podman/usr/local/bin//podman filtering at log level debug
DEBU[0000] Called run.PersistentPreRunE(/home/tobwen/podman/usr/local/bin//podman --conmon /home/tobwen/podman/usr/local/bin//conmon --network-cmd-path /home/tobwen/podman/usr/local/bin//slirp4netns --runtime /home/tobwen/podman/usr/local/bin//crun --storage-driver overlay --storage-opt overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs --log-level debug run example)
DEBU[0000] overlay storage already configured with a mount-program
DEBU[0000] overlay storage already configured with a mount-program
DEBU[0000] overlay storage already configured with a mount-program
DEBU[0000] Using conmon: "/home/tobwen/podman/usr/local/bin//conmon"
DEBU[0000] Initializing boltdb state at /home/tobwen/.local/share/containers/storage/libpod/bolt_state.db
DEBU[0000] Using graph driver overlay
DEBU[0000] Using graph root /home/tobwen/.local/share/containers/storage
DEBU[0000] Using run root /run/user/1000/containers
DEBU[0000] Using static dir /home/tobwen/.local/share/containers/storage/libpod
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp
DEBU[0000] Using volume path /home/tobwen/.local/share/containers/storage/volumes
DEBU[0000] overlay storage already configured with a mount-program
DEBU[0000] Set libpod namespace to ""
DEBU[0000] [graphdriver] trying provided driver "overlay"
DEBU[0000] overlay: mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs
DEBU[0000] backingFs=extfs, projectQuotaSupported=false, useNativeDiff=false, usingMetacopy=false
DEBU[0000] Initializing event backend journald
DEBU[0000] configured OCI runtime crun initialization failed: no valid executable found for OCI runtime crun: invalid argument
DEBU[0000] configured OCI runtime runc initialization failed: no valid executable found for OCI runtime runc: invalid argument
DEBU[0000] configured OCI runtime kata initialization failed: no valid executable found for OCI runtime kata: invalid argument
DEBU[0000] Using OCI runtime "/home/tobwen/podman/usr/local/bin//crun"
INFO[0000] Found CNI network podman (type=bridge) at /home/tobwen/.config/cni/net.d/87-podman.conflist
INFO[0000] Found CNI network cni-podman1 (type=bridge) at /home/tobwen/.config/cni/net.d/cni-podman1.conflist
INFO[0000] Found CNI network cni-podman (type=bridge) at /home/tobwen/.config/cni/net.d/cni-podman.conflist
DEBU[0000] Default CNI network name podman is unchangeable
INFO[0000] Setting parallel job count to 25
DEBU[0000] Looking up image "example" in local containers storage
DEBU[0000] Trying "example" ...
DEBU[0000] Loading registries configuration "/home/tobwen/.config/containers/registries.conf"
DEBU[0000] Trying "localhost/example:latest" ...
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] Found image "example" as "localhost/example:latest" in local containers storage
DEBU[0000] Looking up image "example" in local containers storage
DEBU[0000] Trying "example" ...
DEBU[0000] Trying "localhost/example:latest" ...
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] Found image "example" as "localhost/example:latest" in local containers storage
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] Found image "example" as "localhost/example:latest" in local containers storage ([overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5)
DEBU[0000] Inspecting image 9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@6dbb9cc54074106d46d4ccb330f2a40a682d49dda5f4844962b7dce9fe44aaec"
DEBU[0000] exporting opaque data as blob "sha256:6dbb9cc54074106d46d4ccb330f2a40a682d49dda5f4844962b7dce9fe44aaec"
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@a61a0863d0be1fff22df2e46247590bec9ee4f39854d8ac83455c2308d00ddee"
DEBU[0000] exporting opaque data as blob "sha256:a61a0863d0be1fff22df2e46247590bec9ee4f39854d8ac83455c2308d00ddee"
DEBU[0000] Looking up image "example" in local containers storage
DEBU[0000] Trying "example" ...
DEBU[0000] Trying "localhost/example:latest" ...
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] Found image "example" as "localhost/example:latest" in local containers storage
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] Found image "example" as "localhost/example:latest" in local containers storage ([overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5)
DEBU[0000] Inspecting image 9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@6dbb9cc54074106d46d4ccb330f2a40a682d49dda5f4844962b7dce9fe44aaec"
DEBU[0000] exporting opaque data as blob "sha256:6dbb9cc54074106d46d4ccb330f2a40a682d49dda5f4844962b7dce9fe44aaec"
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@a61a0863d0be1fff22df2e46247590bec9ee4f39854d8ac83455c2308d00ddee"
DEBU[0000] exporting opaque data as blob "sha256:a61a0863d0be1fff22df2e46247590bec9ee4f39854d8ac83455c2308d00ddee"
DEBU[0000] Inspecting image 9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5
DEBU[0000] Image has volume at "oops"
DEBU[0000] Adding anonymous image volume at "oops"
DEBU[0000] using systemd mode: false
DEBU[0000] No hostname set; container's hostname will default to runtime default
DEBU[0000] Loading seccomp profile from "/usr/share/containers/seccomp.json"
DEBU[0000] Allocated lock 10 for container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52
DEBU[0000] parsed reference into "[overlay@/home/tobwen/.local/share/containers/storage+/run/user/1000/containers:overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs]@9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] exporting opaque data as blob "sha256:9a979abb1727ad4820524fcb6ac30889fb2bf8f336878b2097cae4f2857479b5"
DEBU[0000] created container "edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52"
DEBU[0000] container "edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52" has work directory "/home/tobwen/.local/share/containers/storage/overlay-containers/edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52/userdata"
DEBU[0000] container "edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52" has run directory "/run/user/1000/containers/overlay-containers/edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52/userdata"
DEBU[0000] Creating new volume 75a8448bf9f20c7b7e97016d5c1b6006e4e9d559ac441ae6c5b782507009a7c0 for container
DEBU[0000] Validating options for local driver
DEBU[0000] Not attaching to stdin
DEBU[0000] [graphdriver] trying provided driver "overlay"
DEBU[0000] overlay: mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs
DEBU[0000] backingFs=extfs, projectQuotaSupported=false, useNativeDiff=false, usingMetacopy=false
DEBU[0000] overlay: mount_data=,lowerdir=/home/tobwen/.local/share/containers/storage/overlay/l/CCNGKXP2PNCHKRZSNEKI3ZU43W,upperdir=/home/tobwen/.local/share/containers/storage/overlay/04ef0655b0a2e527171de714ea6c2916b06bd29c2efdc5a953678cfbf75827bd/diff,workdir=/home/tobwen/.local/share/containers/storage/overlay/04ef0655b0a2e527171de714ea6c2916b06bd29c2efdc5a953678cfbf75827bd/work
DEBU[0000] Made network namespace at /run/user/1000/netns/cni-b7531307-95ff-a388-4d0b-aa025951e283 for container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52
DEBU[0000] mounted container "edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52" at "/home/tobwen/.local/share/containers/storage/overlay/04ef0655b0a2e527171de714ea6c2916b06bd29c2efdc5a953678cfbf75827bd/merged"
DEBU[0000] Going to mount named volume 75a8448bf9f20c7b7e97016d5c1b6006e4e9d559ac441ae6c5b782507009a7c0
DEBU[0000] slirp4netns command: /home/tobwen/podman/usr/local/bin//slirp4netns --disable-host-loopback --mtu=65520 --enable-sandbox --enable-seccomp -c -e 3 -r 4 --netns-type=path /run/user/1000/netns/cni-b7531307-95ff-a388-4d0b-aa025951e283 tap0
DEBU[0000] Copying up contents from container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 to volume 75a8448bf9f20c7b7e97016d5c1b6006e4e9d559ac441ae6c5b782507009a7c0
DEBU[0000] Created root filesystem for container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 at /home/tobwen/.local/share/containers/storage/overlay/04ef0655b0a2e527171de714ea6c2916b06bd29c2efdc5a953678cfbf75827bd/merged
DEBU[0000] Workdir "/" resolved to host path "/home/tobwen/.local/share/containers/storage/overlay/04ef0655b0a2e527171de714ea6c2916b06bd29c2efdc5a953678cfbf75827bd/merged"
DEBU[0000] /etc/system-fips does not exist on host, not mounting FIPS mode subscription
DEBU[0000] Setting CGroups for container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 to user.slice:libpod:edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52
DEBU[0000] reading hooks from /usr/share/containers/oci/hooks.d
DEBU[0000] Created OCI spec for container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 at /home/tobwen/.local/share/containers/storage/overlay-containers/edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52/userdata/config.json
DEBU[0000] /home/tobwen/podman/usr/local/bin//conmon messages will be logged to syslog
DEBU[0000] running conmon: /home/tobwen/podman/usr/local/bin//conmon  args="[--api-version 1 -c edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 -u edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 -r /home/tobwen/podman/usr/local/bin//crun -b /home/tobwen/.local/share/containers/storage/overlay-containers/edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52/userdata -p /run/user/1000/containers/overlay-containers/edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52/userdata/pidfile -n hungry_chaplygin --exit-dir /run/user/1000/libpod/tmp/exits --full-attach -s -l k8s-file:/home/tobwen/.local/share/containers/storage/overlay-containers/edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52/userdata/ctr.log --log-level debug --syslog --conmon-pidfile /run/user/1000/containers/overlay-containers/edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52/userdata/conmon.pid --exit-command /home/tobwen/podman/usr/local/bin/podman --exit-command-arg --root --exit-command-arg /home/tobwen/.local/share/containers/storage --exit-command-arg --runroot --exit-command-arg /run/user/1000/containers --exit-command-arg --log-level --exit-command-arg debug --exit-command-arg --cgroup-manager --exit-command-arg systemd --exit-command-arg --tmpdir --exit-command-arg /run/user/1000/libpod/tmp --exit-command-arg --runtime --exit-command-arg /home/tobwen/podman/usr/local/bin//crun --exit-command-arg --storage-driver --exit-command-arg overlay --exit-command-arg --storage-opt --exit-command-arg overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs --exit-command-arg --events-backend --exit-command-arg journald --exit-command-arg --syslog --exit-command-arg container --exit-command-arg cleanup --exit-command-arg edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52]"
INFO[0000] Running conmon under slice user.slice and unitName libpod-conmon-edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52.scope
[conmon:d]: failed to write to /proc/self/oom_score_adj: Permission denied

DEBU[0000] Received: 5001
INFO[0000] Got Conmon PID as 4998
DEBU[0000] Created container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 in OCI runtime
DEBU[0000] Attaching to container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52
DEBU[0000] Starting container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52 with command [/bin/sh]
DEBU[0000] Started container edda5c6ef45a76369aa02cb300dca695458a4b0df9f5480e998158da24e56b52
DEBU[0000] Enabling signal proxying
DEBU[0000] Called run.PersistentPostRunE(/home/tobwen/podman/usr/local/bin//podman --conmon /home/tobwen/podman/usr/local/bin//conmon --network-cmd-path /home/tobwen/podman/usr/local/bin//slirp4netns --runtime /home/tobwen/podman/usr/local/bin//crun --storage-driver overlay --storage-opt overlay.mount_program=/home/tobwen/podman/usr/local/bin//fuse-overlayfs --log-level debug run example
$
```

</details>

``` shell
$ podman ps -a
CONTAINER ID  IMAGE                          COMMAND               CREATED        STATUS                    PORTS   NAMES
edda5c6ef45a  localhost/example:latest       /bin/sh               3 minutes ago  Exited (0) 3 minutes ago          hungry_chaplygin
```

**Comment by Luap99** [comment]
Yes this should be fixed in v3.2

**Comment by ecnepsnai** [comment]
Thanks @Luap99 !
